tasks/main.yml

```diff
@@ -14,7 +14,34 @@
     # it.
     state: latest  # noqa 403
     update_cache: yes
-  when: ansible_os_family == "Debian"
+  when: >
+    ansible_os_family == "Debian" and
+    ansible_distribution_release != "kali-rolling"
+
+- name: Reinstall cryptsetup-initramfs so upgrade doesn't fail
+  apt:
+    name: 'cryptsetup-initramfs'
+    state: latest  # noqa 403
```

Comment:
Please add a bit more explanation to your comment above about why we want to use `latest` here and above.

Reply:
Addressed in commit 9f51229